### ISSUE TITLE
[bug] duplicated default export with multiple wildcast exports

### ISSUE THREAD

**Issue by TechQuery** [parcel-bundler/parcel#7483]
### `package.json`

```json
{
  "browserslist": "> 0.5%, last 2 versions, not dead",
  "targets": {
    "main": {
      "optimize": true
    }
  },
  "scripts": {
    "build": "rm -rf dist/  &&  parcel build"
  }
}
```

### `tsconfig.json`

```typescript
{
    "compilerOptions": {
        "esModuleInterop": true,
        "jsx": "react-jsx",
        "skipLibCheck": true,
        "lib": ["ES2021", "DOM"]
    },
    "include": ["source/**/*"]
}
```

### `.parcelrc`

```json
{
  "extends": "@parcel/config-default",
  "transformers": {
    "*.{ts,tsx}": ["@parcel/transformer-typescript-tsc"]
  }
}
```

## 🤔 Expected Behavior

compile `index.ts`:

```typescript
export * from "./A";
export * from "./B";
```

to `index.esm.js`

```javascript
export { A$export$xxx, B$export$yyy };
```

## 😯 Current Behavior

```javascript
export { A$export$xxx, A$exports as default, B$export$yyy, B$exports as default };
```

## 🌍 Your Environment

| Software         | Version(s)          |
| ---------------- | ------------------- |
| Parcel           | 2.0.1               |
| Node             | 14.17.6             |
| NPM              | 6.14.15             |
| Yarn             | 1.22.15             |
| Operating System | Windows 10 Pro 21H1 |

**Comment by TechQuery** [comment]
Anyone knows something?

**Comment by TechQuery** [comment]
This error is fixed at least in `parcel@2.6.2`: idea2app/Idea-React@b63fd6c
